dts/bindings/ethernet/realtek,rtl8211f.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023-2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: Realtek RTL8211F Ethernet PHY device
+
+compatible: "realtek,rtl8211f"
+
+include: ethernet-phy.yaml
+
+properties:
+  rt,reset-gpio:
+    type: phandle-array
+    specifier-space: gpio
+    description: GPIO connected to PHY reset signal pin. Reset is active low.
+  rt,interrupt-gpio:
```

Comment:
since these properties are on other phys, I wonder if we can generalize them (not required, but a suggestion)